sign/Makefile.am

```diff
@@ -3,7 +3,8 @@
 include $(top_srcdir)/rpm.am
 AM_CFLAGS = @RPMCFLAGS@
 
-AM_CPPFLAGS = -I$(top_builddir) -I$(top_srcdir) -I$(top_builddir)/include/
+AM_CPPFLAGS = -I$(top_builddir) -I$(top_srcdir) -I$(top_builddir)/include/ \
+	-I$(includedir)
```

Comment:
This isn't right, $(includedir) reflects where includes of this build of rpm is to be installed on "make install". What problem is this change trying to solve?

Reply:
I was using this to include headers from a package that wasn't installed on my system in order to build rpm against it. In particular before I had a working fsverity-utils package providing libfsverity.h installed.

I can drop this if you don't like it.

Reply:
Understood, the right way to do that is to pass CPPFLAGS=-I/some/path as an argument to ./configure. So yeah please drop it.

Reply:
It's gone